pom.client.xml

```diff
@@ -703,6 +703,6 @@
     <module>./eventhubs/client</module>
     <module>./keyvault/client</module>
     <module>./sdk/tracing</module>
-    <module>./identity/client</module>
+    <module>./sdk/identity/azure-identity</module>
```

Comment:
So, we are not using `client` for track 2 sdks @JonathanGiles

Reply:
Unfortunately not. The pattern is `microsoft-azure-` is track one, `azure-` is track two.

Reply:
Yes we are trying to use the package name and not client or data-plane monikers